tests/admin_views/test_multidb.py

```diff
@@ -0,0 +1,61 @@
+from unittest.mock import patch
```

Comment:
While we support Python 2, use `from django.test import mock`. Also, you may squash your commits when updating.